jenkins/ompi/test_env.sh

```diff
@@ -0,0 +1,407 @@
+export OMPI_MCA_mpi_warn_on_fork='0'
```

Comment:
why this file needed?

Reply:
I don't actually remember adding that. Let me look into it.